ush/module-setup.sh

```diff
@@ -52,7 +52,7 @@ elif [[ ${MACHINE_ID} = s4* ]] ; then
 elif [[ ${MACHINE_ID} = wcoss2 ]]; then
     # We are on WCOSS2
     # Ignore default modules of the same version lower in the search path (req'd by spack-stack)
-    #export LMOD_TMOD_FIND_FIRST=yes #TODO: Uncomment this when using spack-stack
+    export LMOD_TMOD_FIND_FIRST=yes
```

Comment:
This should only be uncommented when spack-stack is used for the entire workflow.  Currently, it is only used for the GDASApp and using this flag for other components will cause issues.  Adding a suggestion to the comment to clarify this:

```suggestion
    #export LMOD_TMOD_FIND_FIRST=yes #TODO: Uncomment this when using spack-stack for the entire workflow
```

Reply:
oh I didn't realize that was the case, I'll commit this suggestion